content/en/community/hackathons/2023-04-amsterdam/challenges/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Challenges
+date: 2023-04-13T05:27:37+10:00
```

Comment:
You can change the date to `04-16` (or whatever the day of the hackathon is) to hide this page until the day of the challenges.
You can also leave it like this if you want.